vhdl_lang/src/syntax/expression.rs

```diff
@@ -1140,6 +1140,13 @@ mod tests {
 
         assert_expression_is("1-2-3", "((Integer(1) Minus Integer(2)) Minus Integer(3))");
 
```

Comment:
This fix should really have been a separate PR. It creates value on its own and should not wait for the other stuff.

Reply:
Agreed, I'll split it off.